src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphEntityDAO.scala

```diff
@@ -71,6 +71,7 @@ class GraphEntityDAO extends EntityDAO with GraphDAO {
       workspace.addEdge(entity.entityType, newVertex)
       newVertex
     })
+    entityVertex.setProperty("_vaultId", entity.vaultId)
```

Comment:
I remember reading that Orient had a problem with null properties - @mattsooknah do you know if property value being null is a problem?